<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreAjoutIngredient</class>
 <widget class="QDialog" name="FenetreAjoutIngredient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: lightgray;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="2" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="sb_quantite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>g</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>kg</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cb_type">
       <item>
        <property name="text">
         <string>Autre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Boisson</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Légume</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fruit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Viande</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poisson</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fromage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entrée</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dessert</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label4">
       <property name="text">
        <string>A consommer avant le :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Quantité :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_nom"/>
     </item>
     <item row="3" column="3">
      <widget class="QDateEdit" name="de_date"/>
     </item>
     <item row="5" column="3">
      <widget class="QComboBox" name="cb_image"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label5">
       <property name="text">
        <string>Image :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_nom</tabstop>
  <tabstop>cb_type</tabstop>
  <tabstop>sb_quantite</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>de_date</tabstop>
  <tabstop>cb_image</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
